lib/tpm2_session.c

```diff
@@ -94,6 +94,10 @@ void tpm2_session_set_path(tpm2_session_data *data, const char *path) {
     data->path = path;
 }
 
+TPMI_ALG_HASH tpm2_session_data_get_authhash(tpm2_session_data *data) {
+    return data->auth_hash;
+}
+
```

Comment:
@mayuanchenma is this needed?

Reply:
struct tpm2_session_data is defined in tpm2_session.c, cannot get elements of the struct tpm2_session_data in other files.